language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs

```diff
@@ -126,19 +139,27 @@ tests damlc daml2ts davl = testGroup "daml2Ts"
       buildProject charliesRestaurant []
       assertBool "'charlies-restaurant-1.0.dar' was not created." =<< doesFileExist charliesRestaurantDar
       step "Generating TypeScript of 'charlies-restaurant'..."
-      daml2tsProject [charliesRestaurantDar] charliesRestaurantTs
-      assertBool "'CharliesRestaurant.ts' was not created." =<< doesFileExist (charliesRestaurantTs </> "charlies-restaurant-1.0" </> "CharliesRestaurant.ts")
-      assertBool "'packageId.ts' was not created." =<< doesFileExist (charliesRestaurantTs </> "charlies-restaurant-1.0" </> "packageId.ts")
-      assertBool "'Grover.ts' was created." . not =<< doesFileExist (charliesRestaurantTs </> "grover-1.0" </> "Grover.ts")
+      daml2tsProject [charliesRestaurantDar] (charliesRestaurantTs </> "src")
+      assertBool "'CharliesRestaurant.ts' was not created." =<< doesFileExist (charliesRestaurantTs </> "src" </> "charlies-restaurant-1.0" </> "CharliesRestaurant.ts")
+      assertBool "'packageId.ts' was not created." =<< doesFileExist (charliesRestaurantTs </> "src" </> "charlies-restaurant-1.0" </> "packageId.ts")
+      assertBool "'Grover.ts' was created." . not =<< doesFileExist (charliesRestaurantTs </> "src" </> "grover-1.0" </> "Grover.ts")
```

Comment:
```suggestion
      assertBool "'Grover.ts' was created." . not =<< doesFileExist (charliesRestaurantTs </> "src" </> "grover-1.0" </> "Grover.ts")
```
```suggestion
      assertBool "'Grover.ts' was  not created." . not =<< doesFileExist (charliesRestaurantTs </> "src" </> "grover-1.0" </> "Grover.ts")
```

Reply:
No, it's right as is I think. The logic is reversed. We are testing that the file has not been created. This is because the `Grover.dar` has not been included so `Grover.ts` is under it's package name, not the main-package name `grover-1.0` at this point.